pyproject.toml

```diff
@@ -238,3 +238,9 @@ enable_error_code = ['explicit-override']
 ignore_missing_imports = true
 warn_redundant_casts = true
 warn_unused_ignores = true
+
+[tool.codespell]
```

Comment:
It's not required for the documentation, but I think it makes sense to add `codespell` to the `doc` dependency group so one can easily run the same version of codespell as everybody else.